agent/qrexec-agent.c

```diff
@@ -120,6 +126,86 @@ static struct pam_conv conv = {
     NULL
 };
 #endif
+
+_Noreturn static void really_exec(const char *prog, const struct passwd *pw,
+                                  char **env, const char *cmd, pid_t agent_pid)
+{
+    /* 1 means init, which is completely wrong. */
+    if (agent_pid < 2) {
```

Comment:
Why is it wrong? I can imagine legitimate case of using qrexec as init (maybe some version of a stubdomain, or some other very specialized domain?